TGM/src/main/java/network/warzone/tgm/modules/tdm/TDMModule.java

```diff
@@ -99,7 +99,7 @@ public void onScoreboardInit(ScoreboardInitEvent event) {
     }
 
     private String getTeamScoreLine(MatchTeam matchTeam) {
-        return "  " + ChatColor.RESET + pointsModule.getPoints(matchTeam) + ChatColor.GRAY + "/" + pointsModule.getTarget(matchTeam) + ChatColor.RESET + " Kills";
+        return "  " + ChatColor.RESET + pointsModule.getPoints(matchTeam) + ChatColor.GRAY + "/" + pointsModule.getTarget(matchTeam) + ChatColor.WHITE + " Kills";
```

Comment:
Why change this? White and reset aren't the same thing.

Reply:
Changed that because for some reason this was happening
![weird colors](https://i.imgur.com/tGIE6F0.png)

Reply:
doesnt it break now when a team has like 3 Objectives.
like on some maps theres 2 teams with 3 objectives (CTW)